Author: Montel Laurent <montel@kde.org>
Description: Switch the "Reply using HTML if present" option as off by default
Origin: upstream, commit:fe8ab0c4b6bebe0e5806a870b0946e13bec92c7e
Bug-Debian: http://bugs.debian.org/717113
Applied-Upstream: 4.11

--- a/templateparser/templateparser.kcfg
+++ b/templateparser/templateparser.kcfg
@@ -18,7 +18,7 @@ xsi:schemaLocation="http://www.kde.org/s
       </entry>
         <entry name="ReplyUsingHtml" type="Bool" key="replySameFormat">
         <label>Reply using HTML if present</label>
-        <default>true</default>
+        <default>false</default>
       </entry>
 
       <entry name="CustomTemplates" type="StringList" key="CustomTemplates" />
